### DISCUSSION TITLE
Render media in Lexical Editor

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#8014] [Feature Requests & Ideas]
Following up from #5072 , it would be great if the Lexical editor has support for rendering inline media / blocks in the editor, so that we can resize / move the media / blocks via dragging.

Example: https://github.com/infonomic/lexical-inline-image-plugin

![image](https://github.com/user-attachments/assets/2e4ce8e8-7fc5-45dd-85c4-09ff404990d2)

**Comment by sebastianmulders** [comment]
Has there been any update on this? I know @jmikrut mentioned that this can be achieved using inlineBlocks, but I’ve tried various approaches and haven’t been able to get anything solid working. Would love to hear if there are any examples or best practices for this. Thanks!

**Reply by tyteen4a03** [reply]
I hope to see this in the mainline editor one day, but I think for now you'll need a custom Lexical feature for this.
